config.toml.sample

```diff
@@ -19,36 +19,44 @@ graphql_url = "http://127.0.0.1:3001"
 source = "neutron"
 rpc = "https://rpc-kralum.neutron-1.neutron.org:443/"
 jitter = 10000
-pagination_limit = 10000
+limit = 10000
+[protocols.neutron.assets.dATOM]
+multiplier = 1
+denom = "ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9"
+
+[protocols.neutron.frontend_data]
 link = "https://www.neutron.org"
 link_text = "Go to Neutron"
 chain_name = "neutron"
-[protocols.neutron.assets.dATOM]
+[protocols.neutron.assets.dATOM.frontend_data]
+visible = true
 strategy = "dATOM"
 description = "Holding dATOM on Neutron"
 type = "single-sided"
-multiplier = 1
+featured = false
 status = true
-denom = "ibc/C4CFF46FD6DE35CA4CF4CE031E643C8FDC9BA4B99AE598E9B0ED98FE3A2319F9"
-
 
 [protocols.osmosis-levana]
 rpc = "https://osmosis-rpc.polkachu.com:443/"
 source = "levana"
 jitter = 10000
 concurrency_limit = 3
 pagination_limit = 300
+[protocols.osmosis-levana.assets.dATOM]
+multiplier = 2
+lp_token = "osmo1kp5nhg2eqcv8zwc0ndgjj7w8pk8dzhu58jfk4q0ptfywj2m42qvqhh9dcv"
+
+[protocols.osmosis-levana.frontend_data]
```

Comment:
For some protocols naming schema is `"protocols.protocol_name.frontend_data"`, for others it is `"protocols.protocol_name.assets.asset_name.frontend_data"`. Please confirm it is not a mistake.

Reply:
It is not a mistake, it is because we have both some frontend data for the whole protocol itself and frontend data for the certain asset